abdulfataiaka/pluralsight-admin
# #161346730 Hide empty course list

#### What does this PR do
- Hide empty course list

#### Description of work done
- Show empty content text when there are no courses

#### How should it be manually tested
- Clone the repository `git clone https://github.com/abdulfataiaka/pluralsight-admin.git`
- Checkout to this branch
- Delete all course from the courses page
- Ensure you see empty content text

#### Relevant pivotal tracker story
[161345849](https://www.pivotaltracker.com/story/show/161345849)

#### Screenshots
<img width="1440" alt="screen shot 2018-10-19 at 4 09 46 pm" src="https://user-images.githubusercontent.com/32123313/47226844-6c0cf880-d3b9-11e8-8746-22d9d597bc43.png">
